build_daemon.sh

```diff
@@ -0,0 +1,2 @@
+cd packages/pipboard && npm run build && mkdir -p ../daemon/src/app/public && cp -r ./build/. ../daemon/src/app/public \
+&& cd ../cli/assets && mkdir -p server && rsync -av --progress ../../daemon/. server --exclude=node_modules
```

Comment:
To build daemon, pipboard should not be required, and this script should be also moved to packages/daemon.

Reply:
Deamon should also be used as a server to serve pipboard. This one is used to build pipboard and put the built pipboard into public (serve) folder under daemon.

Reply:
Thus IMO we need to add @pipcook/pipboard to daemon's dependencies, and do what this installation in daemon itself :)